.github/workflows/integration.yml

```diff
@@ -21,14 +21,19 @@ jobs:
       run: |
         git clone https://github.com/open-contracting/cove-${{ matrix.cove }}.git
         git clone https://github.com/open-contracting/lib-cove-${{ matrix.cove }}.git
+        git clone https://github.com/OpenDataServices/lib-cove-web.git
 
     - name: Install
       run: |
         pip install -r requirements_dev.txt
+        # Temporary fix so we don't try to install libcove from a non-existent
+        # branch
+        sed -i '/=libcove/d' cove-${{ matrix.cove}}/requirements_dev.txt
```

Comment:
What's this about?

Reply:
This is because cove-ocds pulls in a fork of this repo that no longer exists. We install it as part of our integration tests, but can delete the line from requirements.txt as we explicitly install this version of the library.